python-avd/pyavd/_eos_designs/structured_config/base/__init__.py

```diff
@@ -663,8 +663,7 @@ def ptp(self) -> dict | None:
             if get(self._hostvars, "ptp") is not None:
```

Comment:
```suggestion
            # TODO: AVD5.0.0 Remove this handling since the `ptp` key is removed from eos_designs.
            if get(self._hostvars, "ptp") is not None:
```